### ISSUE TITLE
While navigating using keyboard, keyboard focus color contrast failed on "1BEAE2B0-134A-4780-9A7A-5FA17EADD513" table data Which is having"2.416:1" ratio which is less than 3:1: A11y_Azure Databases extension for VS Code_Non-Text Contrast

### ISSUE THREAD

**Issue by PadminiGariki** [microsoft/vscode-cosmosdb#2927]
### GitHub Tags:
#A11ySev2; #A11yTCS; #DesktopApp; #BM_Azure Databases extension for VS Code_Win32_Nov2025; #Win32; #Win11; #FTP; #A11yMAS; #Azure Databases extension for VS Code; #Benchmark; #Non-Text Contrast; #A11YWCAG2.1; #MAS1.4.11; 

"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification."

"[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### Environment Details:
Application Name: Azure Databases extension for VS Code
OS Version: Windows 11 enterprise 24H2. (OS build 26120.3360)

### Steps to Reproduce

1.Open VS Code
2.Open the "Azure" Sidebar
3.Sign In to your Azure Account (skip if already signed in)
4.Expand the "Cosmosdb-Portalteam-generaltest-msft" subscription node
5.Expand the "Azure Cosmos DB" node
6.Expand the "a11y-nosql" Cosmos DB account
7.Expand the "SampleDB" database node
8.Expand the "SampleContainer" node.
9.Tab till "Create query" and press enter.
10.Run a simple query such as "SELECT TOP 1000 * FROM c"
11.Tab till "Results" tab and press enter.
12.Tab till "Table data".
13.Observed the issue While navigating using keyboard, keyboard focus color contrast failed on "1BEAE2B0-134A-4780-9A7A-5FA17EADD513" table data Which is having"2.416:1" ratio which is less than 3:1.

### Actual Results:
While navigating using keyboard, keyboard focus color contrast failed on "1BEAE2B0-134A-4780-9A7A-5FA17EADD513" table data Which is having"2.416:1" ratio which is less than 3:1.

### Expected Results:
While navigate using keyboard, keyboard focus on "1BEAE2B0-134A-4780-9A7A-5FA17EADD513" table data color contrast should meet the ratio >=3:1.

### User Impact:
Users with low vision and with color vision deficiency will find it difficult to interact with the non-textual content which have a low contrast ratio.

### Attachment:

<img width="1226" height="623" alt="Image" src="https://github.com/user-attachments/assets/db64faf4-1b9f-462c-bb52-254f62c19f80" />

**Comment by PadminiGariki** [comment]
As per above comment, close this bug does not repro after applying High contrast theme.
For reference, attachment is added below.
#Closed; #Regressed:02-18-26;

<img width="1365" height="716" alt="Image" src="https://github.com/user-attachments/assets/79d6c4c3-5201-489e-a105-67fd4a92dc95" />
